### ISSUE TITLE
Browserify require('typescript) error on windows

### ISSUE THREAD

**Issue by DavidGriswoldTeacher** [microsoft/TypeScript#33224]
I do not know how to make a self-contained report of this bug, because I don't really know what's causing it. The code below is the *javascript* I'm using that is creating the bug.

```js
let ts = require("typescript");
...
let tsAnswer = ts.transpile(answer);
```

**Expected behavior:**
I can transpile code for an online playground-type thing (specifically a typescript version of CodingJS from https://github.com/the-winter/codingjs
**Actual behavior:**
I get an error and can't even open the page. The error is:
`TypeError: ts.perfLogger.logInfoEvent is not a function`

**Platform** Windows 10

I was able to solve this problem by downgrading typescript in the project to version 3.5.3. I suspect the issue comes from #32612 . It's possible I need to add a new package, but the code is not working as is, which is a bug.

**Comment by weswigham** [comment]
It's a fault on our part - adding `"@microsoft/typescript-etw": false` into our `browser` field does prevent the bundler from attempting to bundle it, but rather than returning `undefined` or throwing like one may expect on a `require` failure, it just returns [an empty object](https://github.com/defunctzombie/package-browser-field-spec#ignore-a-module). 🤦‍♀ 

The fix should be easy for us.

**Comment by asakhuj** [comment]
Hi ! I am still facing the issue ! My project was working fine few days back. Now I am getting this error on browser 

TypeError: ts.perfLogger.logInfoEvent is not a function
(anonymous function)
/node_modules/typescript/lib/typescript.js:3418
  3415 | 
  3416 | 
  3417 |   ts.perfLogger = etwModule ? etwModule : nullLogger;
> 3418 |   ts.perfLogger.logInfoEvent("Starting TypeScript v" + ts.versionMajorMinor + " with command line: " + JSON.stringify(process.argv));
  3419 | })(ts || (ts = {}));
  3420 | /* @internal */
  3421 |

**Comment by weswigham** [comment]
What TS version are you using? This fix went out in the `3.6.3` patch.

**Comment by asakhuj** [comment]
I was using 3.6.2 ! Upgraded to 3.6.3 now ! It's working. Thank you
